package.json

```diff
@@ -90,13 +90,15 @@
     "@sharegate/stylelint-config-recommended": "2.1.1",
     "@storybook/addon-a11y": "6.5.13-alpha.0",
     "@storybook/addon-essentials": "6.5.13-alpha.0",
+    "@storybook/addon-interactions": "^6.5.13",
```

Comment:
Is it required?

I tryed this SB feature lately and I doubt this is something we want to invest into at the moment.

Reply:
We had an issue with remainingAvatar that needed to be tested via interactions. On hover the remaining avatars appears and they were not styled properly.